src/components/item_card/Presenter.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import './Presenter.scss';
+
+const ItemCard = ({ item }) => {
+    let { title, price, image, end_date } = item;
+    const dateFormatter = (endDate) => {
```

Comment:
Are there any tests for this?

Reply:
We extracted this out and added tests in the `show_item` PR